guides/plugins/apps/clientside-to-app-backend.md

```diff
@@ -0,0 +1,136 @@
+# Client side communication to the app backend
+
+To give some context to the request from the browser directly to the app backend, we allow generating a JWT token that contains session information as claims and is signed by the shop.
+
+{% hint style="warning" %}
+The JWT key can be only generated when in the browser the user is logged-in.
+{% endhint %}
+
+## The Flow
+
+```mermaid
+sequenceDiagram
+    participant Client
+    participant Shopware Backend
+    participant App Server
+    Client->>Shopware Backend: POST /store-api/app-system/MyApp/generate-token
+    Shopware Backend->>Client: Responds with Signed JWT Token
+    Client->>App Server: Post /product-review/submit containing JWT in header
+```
```

Comment:
@shyim , Currently, Gibook doesn't support mermaid conversions. Can you please provide a jpeg image as an alternative ?

Reply:
Ignore the above comment. Gitbook now supports mermaid plugin 👍